airflow/models/serialized_dag.py

```diff
@@ -67,7 +67,7 @@ class SerializedDagModel(Base):
     dag_id = Column(String(ID_LEN), primary_key=True)
     fileloc = Column(String(2000), nullable=False)
     # The max length of fileloc exceeds the limit of indexing.
-    fileloc_hash = Column(BigInteger, nullable=False)
+    fileloc_hash = Column(BigInteger().with_variant(Integer, 'sqlite'), nullable=False)
```

Comment:
Similarly, it seems suspicious this needs a variant, but `pickle_hash` on DagPickle does not.

Reply:
It's from the migration files. In some places, we use a particular type for sqlite

Reply:
I wonder if this is intended (can’t think of a reason) or an oversight (seems more likely). The hash shouldn’t use a different integer size…